MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/IReflPresenter.h

```diff
@@ -35,14 +31,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 File change history is stored at: <https://github.com/mantidproject/mantid>.
 Code Documentation is available at: <http://doxygen.mantidproject.org>
 */
-class IReflPresenter {
+class IReflPresenter : public IReflOuterPresenter {
```

Comment:
Do we want this to Inherit Workspace receiver, it looks like it's just passing through. It's still declared abstract (pure virtual) at this scope.